### PR TITLE
Migration from local storage to chrome storage

### PR DESCRIPTION
**Pull request by anto-aligent**
**Description of the proposed changes**
The extension will now use Chrome storage (extension specific) instead of Local storage to store configs

**Screenshots (if applicable)**
_

**Other solutions considered (if any)**
_

**Notes to reviewers**
_

**Time tracking**
Please use Toggl time code **PERT-52: PERT with Wings Code Review**

### CODE REVIEW

File: src/utils/storage.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/**
+ * Retrieve specific data stored against the given key from chrome storage
+ *
+ * @returns { Promise<T> | undefined } data stored in Chrome Storage
+ */
+export const get = async <T>(key: string): Promise<T> => {
+  const dataResult = await chrome.storage.local.get([key]);
```
Comment (thilinaaligent) [inline src/utils/storage.ts:7]: in this file where we use `chrome.` should we change it to `browser.` so it supports both chrome and firefox?

Comment (thilinaaligent) [reply src/utils/storage.ts:7]: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/get

Comment (thilinaaligent) [reply src/utils/storage.ts:7]: nm. looks like we have to set the browser based on what browser we are using

Comment (anto-aligent) [reply src/utils/storage.ts:7]: For now Firefox does not complain when storage is accessed via chrome keyword

**Review by thilinaaligent — APPROVED** [review]
awesome. LGTM

**Comment by thilinaaligent** [comment]
we need to bump the package version to get this deployed. I will create a separate PR for that @anto-aligent

**Comment by anto-aligent** [comment]
@thilinaaligent have added another commit bumping up the package version and also adding chrome types

**Pull request merged**